prometheus_client/exposition.py

```diff
@@ -159,7 +159,39 @@ def _get_best_family(address, port):
     return family, sockaddr[0]
 
 
-def start_wsgi_server(port: int, addr: str = '0.0.0.0', registry: CollectorRegistry = REGISTRY) -> None:
+def _get_ssl_ctx(
+        certfile: str,
+        keyfile: str,
+        protocol: int,
+        cafile: Optional[str] = None,
+        capath: Optional[str] = None,
+        insecure_skip_verify: bool = False,
+) -> ssl.SSLContext:
+    """Load context supports SSL."""
+    ssl_cxt = ssl.SSLContext(protocol=protocol)
+    if cafile is not None:
```

Comment:
I think that with the introduction of `capath`, this condition should be changed to:
```
if cafile or capath:
```

Reply:
I have addressed that in https://github.com/andy-maier/client_python/tree/andy/review-add-tls . Feel free to cherry-pick it from there (just the last commit).

Reply:
Was integrated - resolved.